modules/persistence/README.md

```diff
@@ -0,0 +1,46 @@
+# Snooty Transport Module
+
+As part of integrating Snooty Parser with the unified Snooty toolchain, this module holds exclusive responsibility over transporting and persisting AST and metadata output artifacts from the parser into datastores.
+
+Currently, this module supports persistence to Mongodb Atlas instances, and is also responsible for mutation of entires at persist time.
+
+## Installation
+
+By default, this module requires Node v14.17.6 and uses a `.nvmrc` file if using nvm.
+All that's required for use is running `npm ci`.
+
+## Building and Running
+
+### `npm run build`
+
+Compiles the module using `tsc`. By default, compiles to the `./dist` directory.
```

Comment:
curious, is this a prereq on machine, and how is this handled? (tsc installation)

Reply:
just with `npm i` 👍